iphone/Classes/TiUITableViewProxy.m

```diff
@@ -421,7 +421,7 @@ - (void)updateRow:(id)args
     TiUITableViewAction *action = [[[TiUITableViewAction alloc] initWithObject:newrow animation:anim type:TiUITableViewActionUpdateRow] autorelease];
     [table dispatchAction:action];
   },
-      NO);
+      YES);
```

Comment:
We have some more cases where `dispatchAction` is called directly. Should these be put on a sync-queue as well or are we fine?

Reply:
Ideally it should be. Added. As we discussed we can create a ticket to validate TiThreadPerformOnMainThread in whole sdk.